spec/README.md

```diff
@@ -0,0 +1,58 @@
+# MessageFormat 2.0 Specification
+
+## Table of Contents
+
+1. [Introduction](#introduction)
+   1. [Conformance](#conformance)
+   1. [Terminology and Conventions](#terminology-and-conventions)
+   1. [Design Goals](#design-goals)
+   1. [Design Restrictions](#design-restrictions)
+1. [Syntax](syntax.md)
+   1. [Overview & Examples](syntax.md#overview--examples)
+   1. [Simple Messages](syntax.md#simple-messages)
+   1. [Simple Placeholders](syntax.md#simple-placeholders)
+   1. [Formatting Functions](syntax.md#formatting-functions)
+   1. [Markup Elements](syntax.md#markup-elements)
+   1. [Selection](syntax.md#selection)
+   1. [Local Variables](syntax.md#local-variables)
+   1. [Complex Messages](syntax.md#complex-messages)
+1. [Productions](syntax.md#productions)
+   1. [Message](syntax.md#message)
+   1. [Variable Declarations](syntax.md#variable-declarations)
+   1. [Selectors](syntax.md#selectors)
+   1. [Variants](syntax.md#variants)
+   1. [Patterns](syntax.md#patterns)
+   1. [Placeholders](syntax.md#placeholders)
+   1. [Expressions](syntax.md#expressions)
+   1. [Markup Elements](syntax.md#markup-elements)
+1. [Tokens](syntax.md#tokens)
+   1. [Text](syntax.md#text)
+   1. [Names](syntax.md#names)
+   1. [Quoted Strings](syntax.md#quoted-strings)
+   1. [Escape Sequences](syntax.md#escape-sequences)
+   1. [Whitespace](syntax.md#whitespace)
+1. [Complete EBNF](syntax.md#complete-ebnf)
+1. [Formatting](formatting.md)
+
+## Introduction
```

Comment:
To me this section seems like an introduction to the syntax, rather than the whole spec, especially at it's describing itself as a document that's a pary of the full spec.

Reply:
Good point. See changes.